gitversion/entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -o errexit   # abort on nonzero exitstatus
+set -o pipefail  # don't hide errors within pipes
+
+if [ -z "${GV_GIT}" ]; then
+  echo "No git folder supplied. Using default source '/repo'."
+  GV_GIT=/repo
+fi
```

Comment:
Add check for existence of `.git` directroy.

Reply:
done